tools/amd_build/disabled_features.yaml

```diff
@@ -91,6 +91,31 @@
           "struct curandStateMtgp32*": "curandStateMtgp32*"
         }
       },
+      {
```

Comment:
With this fix [here](https://github.com/ROCmSoftwarePlatform/rocRAND/commit/8875c9b984a9c9d2b8d4a141d347ffff2281d910), we should be able to continue using hiprandStateMtgp32. Would then require fixing the mapping back to curandStateMtgp32 -> hiprandStateMtgp32.

Reply:
That didn't work for me in a first attempt (incomplete type still).